Assets/MixedRealityToolkit.SDK/Experimental/Inspectors/UX/Collections/ScrollingObjectCollectionInspector.cs

```diff
@@ -0,0 +1,357 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please move this to Inspectors/Ux/ScrollingObjectCollection to be consistent with other file locations